packages/utils/test/trimCanvas.tests.ts

```diff
@@ -24,18 +24,25 @@ describe('trimCanvas', () =>
         canvas.width = 100;
         canvas.height = 50;
 
-        context.fillStyle = '#ff0000';
-        context.fillRect(10, 20, 10, 5);
-        context.fillStyle = '#00ff00';
-        context.fillRect(15, 25, 10, 5);
+        if (context !== null)
```

Comment:
We can use:

```ts
if (context === null) fail('Failed to create context');
```

And we don't need to check `context?.xxx` below, since `fail()` returns `never` and TypeScript can infer `context` won't be `null` below.